### ISSUE TITLE
Some Anamolies while using the Trainer

### ISSUE THREAD

**Issue by bvparekh** [sailro/EscapeFromTarkov-Trainer#23]
Hi Sebastian - first of all, thank you creating and updating the trainer. I find it fun and relaxing to use in SPT-AKI version when I just want to crack some heads. The trainer generally works great across different maps with one particular issue that randomly pops up in some maps but is persistent in Interchange (per my experience);

The Occurrence: 100% of the time on Interchange, some times on Labs, Reserve and Customs, infrequently on Shoreline and only once or twice on Factory. 

The Issue: When this issue happens, the wall hack essentially has problem tracking hostiles on the map. You can glimpse some of the hostiles by moving at a particular angle but afterwards, they vanish from the view just to pop up again randomly. It always happens on Interchange every single time I use it. For some reason, the wall hack is not persistent in Interchange. I could see Killa from outside the mall but as soon as I enter, he might disappear and same for scavs. 

I figured, I will flag it to you in case you have any thoughts around why the trainer has issues on Interchange. Other maps, this issue becomes random and I can't figure out what might be causing it. 

Any thoughts or help is appreciated.

**Comment by sailro** [comment]
Yes I spotted this issue as well. Not sure of the root cause yet. Perhaps it is related to the specific geometry/LOD of the map ?

**Comment by bvparekh** [comment]
Appreciate the quick response.

What LOD do you use for testing? My default LOD is 2000. My very basic first reaction was that Interchange, Reserve and Customs had multiple "walls" to see through vs Shoreline or Factory; Not sure if that is creating an issue in interpretation. On Interchange, the hostiles can be viewed from certain angles (easier if you are outside the mall) but disappear if you even move a little. Let me know if I can help with testing since I am not a programmer but I can be a guinea pig.

**Comment by sailro** [comment]
So my guess is that this issue is related to the shader we use to outline enemies:
https://github.com/sailro/EscapeFromTarkov-Trainer/blob/master/Files/FirstOutline.shader

But this is outside of my skillset :)

**Comment by bvparekh** [comment]
No worries. I appreciate all that you are doing (& hope you will continue doing for the SPT-AKI usage); The trainer is fun to use so the issue does not detract from that. Thank you for creating this project!

**Comment by hcwazm** [comment]
can you building just night ，norecoil，stamina，thermal😁

**Comment by hcwazm** [comment]
可以做一个无后座 无限耐力 夜视仪 热成像 的独立版本吗？
